netmanager/src/views/pages/ExportData/index.js

```diff
@@ -1205,8 +1331,7 @@ const ExportData = (props) => {
                 size="small"
                 aria-label="close"
                 color="inherit"
-                onClick={handleClickSnackbar}
-              >
+                onClick={handleClickSnackbar}>
                 <CloseIcon fontSize="small" />
```

Comment:
Experienced this glitch
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/a1afc666-ba56-4005-a106-009125bd3e53)

Reply:
Can we have something informative in this manner an example of a use cause is to inform users that if they want to download bulky volumes of data(or data which is greater than a months worth of data), the schedule option is preferrable to avoid timeouts

![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/9209a178-3bff-47d8-9580-0ee9ae013a31)

Reply:
Hey @Codebmk, thanks for your feedback. The responsiveness still needs some work but will resolve that issue in the next commit. Thanks.

Reply:
`Also we could pick a thing or two on the UI`

Reply:
Resource to guide:
https://www.appcues.com/blog/product-tours-ui-patterns

Reply:
1. Add more details and explanations for the users.
2. Enhance the appearance and functionality of the tooltip.
Noted!!
Thank you for your valuable suggestions, @Codebmk. I will implement them in the next commit.